content/release-10-16-0/announcements-10-16-0.md

```diff
@@ -132,3 +132,15 @@ see the Streaming Analytics release notes for [release 10.16.0](/release-10-16-0
 The following keys for configuring model timeouts have been removed and will be ignored when set:
 `default_timeout_secs` and `block_promise_timeout_secs`.
 Instead, warnings are now logged when request responses or returned promise values take too long to complete.
+
+### Cumulocity IoT DataHub
+
+#### Planned
+
+##### Breaking change in the offloading mechanism - support for mixed types will be deprecated
```

Comment:
##### Breaking change in the offloading mechanism - DEPRECATION OF support for mixed types

Reply:
Done